chat-components/src/components/chatbutton/common/defaultProps/defaultChatButtonControlProps.ts

```diff
@@ -5,10 +5,7 @@ export const defaultChatButtonControlProps: IChatButtonControlProps = {
     id: Ids.DefaultChatButtonId,
     dir: "ltr",
     role: "button",
-    ariaLabel: AriaLabels.LetsChatWeAreOnline,
```

Comment:
Why are these removed?

Reply:
https://dynamicscrm.visualstudio.com/OneCRM/_workitems/edit/3226191 , 

the ask is to remove title and aria label